packages/kununu-theme-v2/scss/base/colors.scss

```diff
@@ -0,0 +1,234 @@
+  background: $gray-base-03;
+}
+
+.gray-base-10 {
+  color: $gray-base-80;
+  background: $gray-base-10;
+}
+
+.gray-base-20 {
+  @extend %text-white;
+  background: $gray-base-20;
+}
+
+.gray-base-30 {
+  @extend %text-white;
+  background: $gray-base-30;
+}
+
+.gray-base-40 {
+  @extend %text-white;
+  background: $gray-base-40;
+}
+
+.gray-base-50 {
+  @extend %text-white;
+  background: $gray-base-50;
+}
+
+.gray-base-60 {
+  @extend %text-white;
+  background: $gray-base-60;
+}
+
+.gray-base-70 {
+  @extend %text-white;
+  background: $gray-base-70;
+}
+
+.gray-base-80 {
+  @extend %text-white;
+  background: $gray-base-80;
+}
+
+.gray-base-90 {
+  @extend %text-white;
+  background: $gray-base-90;
+}
+
+// Colors
+// -------------------------
+
+.green {
+  @extend %text-white;
+  background: $green;
+}
+
+.green-light {
+  @extend %text-white;
+  background: $green-light;
+}
+
+.green-dark {
+  @extend %text-white;
+  background: $green-dark;
+}
+
+.blue {
+  @extend %text-white;
+  background: $blue;
+}
+
+.blue-lighter {
+  @extend %text-white;
+  background: $blue-lighter;
+}
+
+.blue-light {
+  @extend %text-white;
+  background: $blue-light;
+}
+
+.blue-dark {
+  @extend %text-white;
+  background: $blue-dark;
+}
+
+.teal {
+  @extend %text-white;
+  background: $teal;
+}
+
+.red {
+  @extend %text-white;
+  background: $red;
+}
+
+.white {
+  background: $white;
+}
+
+.gray {
+  @extend %text-white;
+  background: $gray;
+}
+
+.gray-light {
+  @extend %text-white;
+  background: $gray-light;
+}
+
+.gray-lightest {
+  background: $gray-lightest;
+}
+
+.facebook-blue {
+  @extend %text-white;
+  background: $facebook-blue;
+}
+
+.twitter-blue {
+  @extend %text-white;
+  background: $twitter-blue;
+}
+
+.google-red {
+  @extend %text-white;
+  background: $google-red;
+}
+
+.youtube-red {
+  @extend %text-white;
+  background: $youtube-red;
+}
+
+.linkedin-blue {
+  @extend %text-white;
+  background: $linkedin-blue;
+}
+
+.xing-green {
+  @extend %text-white;
+  background: $xing-green;
+}
+
+.text-white {
+  @extend %text-white;
+}
+
+.text-gray-light {
+  color: $gray-light;
+}
+
+.text-gray-base-50 {
+  color: $gray-base-50;
+}
+
+.text-gray-base-70 {
+  color: $gray-base-70;
+}
+
+.text-green {
+  color: $green;
+}
+
+.text-green-dark {
+  color: $green-dark;
+}
+
+.text-blue {
+  color: $blue;
+}
+
+.text-red {
+  color: $red;
+}
+
+.text-body {
+  color: $text-color;
+}
+
+.text-headline {
+  color: $gray-base;
+}
+
+.border-green {
+  border: 1px solid $green-dark;
+}
+
+.border-green-light {
+  border: 1px solid $green;
+}
+
+.border-blue {
+  border: 1px solid $blue-dark;
+}
+
+.border-color-gray {
+  border-color: $gray;
+}
+
+.border-color-white {
+  border-color: $white;
+}
+
+.border-color-gray-light {
+  border-color: $gray-light;
+}
+
+.border-color-green {
+  border-color: $green;
+}
+
+.border-color-green-dark {
+  border-color: darken($green-dark, 5%);
+}
+
+.border-color-blue {
+  border-color: $blue-dark;
+}
```

Comment:
Hm, do you think all these colors will ever be used in that extent? I think there are a lot of unnecessary ones that could still blow up our css.

Reply:
Ok, I'll do another look through and remove some more stuff.